src/utilities/utilities.hpp

```diff
@@ -172,13 +193,13 @@ struct Arguments {
   T beta = ConstantOne<T>();
   // Batch-specific arguments
   size_t batch_count = 1;
-  std::vector<size_t> x_offsets; // = {0};
-  std::vector<size_t> y_offsets; // = {0};
-  std::vector<size_t> a_offsets; // = {0};
-  std::vector<size_t> b_offsets; // = {0};
-  std::vector<size_t> c_offsets; // = {0};
-  std::vector<T> alphas; // = {ConstantOne<T>()};
-  std::vector<T> betas; // = {ConstantOne<T>()};
+  std::vector<size_t> x_offsets = {0};
+  std::vector<size_t> y_offsets = {0};
+  std::vector<size_t> a_offsets = {0};
+  std::vector<size_t> b_offsets = {0};
+  std::vector<size_t> c_offsets = {0};
+  std::vector<T> alphas = {ConstantOne<T>()};
+  std::vector<T> betas = {ConstantOne<T>()};
   // Sizes
```

Comment:
Did you have to change this back? This was actually changed to [fix a compilation issue under MSVC 2013](https://github.com/mcian/CLBlast/commit/f7a16d427c11311e80ae5e3a67bc42b6170f6532). So this will have to go back the way it was unfortunately.